rmf_traffic_ros2/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ find_package(rmf_traffic_msgs REQUIRED)
 find_package(rmf_fleet_msgs REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rcpputils REQUIRED)
```

Comment:
Is there a particular motivation to bring in this dependency instead of using the [STL std::getenv](https://en.cppreference.com/w/cpp/utility/program/getenv) function? I suspect `rcpputils::get_env_var` is probably just wrapping that function in `std::string`, but it's easy enough for us to cast from `char*` to `std::string` without bringing in a new dependency.

Granted, this is probably a very easy dependency to satisfy since we're already depending on `rclcpp`, but I usually prefer to err on the side of fewer dependencies when it comes to C++.

Reply:
I was mainly copying what the `rmw_implementation` package does to get the environment variable. I'm happy to change it.